_projects/dendrite.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "Contribute to Dendrite"
+desc: "Dendrite is a work in progress matrix homeserver written in Go, help implement some aspect of the Matrix Federation API"
+difficulty: "Ranges from Easy through to Hard depending on the API!"
+issues:
+ - https://github.com/matrix-org/dendrite/issues/671
+mentors:
+ - erikjohnston
+ - richvdh
```

Comment:
Assume all the mentors here are TBD?

Reply:
Yes, it may also include @babolivier, tbc